modules/nodes-cma-autoscaling-custom-prometheus-config.adoc

```diff
@@ -48,7 +52,27 @@ where:
 +
 <service_account>:: Specifies the name of the service account.
 
-. Use the following command to locate the token assigned to the service account:
+.. Create a `secret` YAML to generate a service account token:
++
+[source,terminal]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: thanos-token
+  annotations:
+    kubernetes.io/service-account.name: thanos
```

Comment:
The instructions above use a variable-type name `<service_account>`, then this line here uses `thanos`. I think it's better if they match, as it may be unclear to the reader that they should match.

Either we should add `<service_account>` here, or change the instructions regarding `oc create serviceaccount <service_account>` to `oc create serviceaccount thanos`.

However, the existing role binding creation instructions further below references `thanos`, I'd suggest just keeping it simple and changing all occurrences of `<service_account>` to `thanos`.

Reply:
outdated